lib/NativeCode/DynamicLibraryLoaderHelper/NativeRender/src/eos_helpers.cpp

```diff
@@ -405,39 +405,29 @@ namespace pew::eos
         }
 
         apply_rtc_options(platform_options);
-        apply_steam_settings(platform_options);
+        // TODO: Re-enable apply_steam_settings(platform_options);
```

Comment:
Is it necessary to have this commented out to unblock this pr?

Reply:
Not certain what you mean by to unblock this pr - but the functionality was turned off because it (the steam integration) wasn't in a test-able state. This will be resolved in a subsequent PR.